config/envs/default.py

```diff
@@ -14,6 +14,8 @@ class DefaultConfig(object):
     FLASK_ROOT = str(Path(__file__).parent.parent.parent)
     FLASK_ENV = environ.get("FLASK_ENV")
 
+    ASSESSMENT_PROCESS_ROLES = environ.get("ASSESSMENT_PROCESS_ROLES", "{}")
```

Comment:
I don't think this being an env var is adding value.
Can we make this a CLI argument? We could still set it as a Secret and pass it in the action and github will obfuscate the value.
Let me know if I've missed something.

Reply:
Done updated to cli arg and ASSESSMENT_PROCESS_ROLES removed. Tests and docs updated